run_runner_locally.sh

```diff
@@ -6,7 +6,10 @@ set -e
 
 # Settings you might want to change
 export PYTHON_BINARY=python3.9
-export PORT=5000
+export PORT=6000  # if you change this, update port_mapping in mirrord.json too
+export LOG_LEVEL=INFO
+#export TRACE_INCOMING_REQUESTS=false
```

Comment:
can we just delete?

Reply:
I want to leave it here so people see and know they can change it easily.